docs/storage/hotplug_volumes.md

```diff
@@ -1,150 +1,237 @@
 # Hotplug Volumes
 
-KubeVirt now supports hotplugging volumes into a running Virtual Machine Instance (VMI). The volume must be either a block volume or contain a disk image. When a VM that has hotplugged volumes is rebooted, the hotplugged volumes will be attached to the restarted VM. If the volumes are persisted they will become part of the VM spec, and will not be considered hotplugged. If they are not persisted, the volumes will be reattached as hotplugged volumes
+KubeVirt supports hotplugging persistent volumes into running Virtual Machines. The volumes may be represented as disks, LUNs, or CD-ROMs.
 
 ## Enabling hotplug volume support
 
 Hotplug volume support must be enabled in the feature gates to be supported. The
 [feature gates](../cluster_admin/activating_feature_gates.md#how-to-activate-a-feature-gate)
-field in the KubeVirt CR must be expanded by adding the `HotplugVolumes` to it.
```

Comment:
Should we add a tiny section about the graduating process of DeclarativeHotplugVolumes/HotplugVolumes precedence?

Reply:
This way we can also subtly "recommend" folks to drop HotplugVolumes

Reply:
added a note on status of `HotplugVolumes` and interaction with `DeclarativeHotplugVolumes`